channel/src/main/java/com/synopsys/integration/alert/channel/jira/cloud/descriptor/JiraCloudDescriptor.java

```diff
@@ -37,8 +38,9 @@
     public static final String JIRA_DESCRIPTION = "Configure the Jira Cloud instance that Alert will send issue updates to.";
 
     @Autowired
-    public JiraCloudDescriptor(JiraCloudGlobalUIConfig globalUIConfig, JiraCloudDistributionUIConfig distributionUIConfig) {
+    public JiraCloudDescriptor(JiraCloudGlobalUIConfig globalUIConfig, JiraCloudDistributionUIConfig distributionUIConfig, JiraCloudGlobalValidator jiraCloudGlobalValidator) {
         super(ChannelKeys.JIRA_CLOUD, distributionUIConfig, globalUIConfig);
+        addGlobalValidator(jiraCloudGlobalValidator);
```

Comment:
I've been trying so hard to get rid of these classes and the anti-pattern of `add...` in the constructor. Can we make an alternate constructor for now with the global validator and get rid of the `add...` at least?